keyboards/karn/readme.md

```diff
@@ -0,0 +1,99 @@
+  * │     │  !  │  @  │  #  │  $  │  %  │             │  ^  │  &  │  *  │  (  │  )  │     │
+  * │     │     │     │     │     │     │             │     │     │     │     │     │     │
+  * └─────┼─────┼─────┼─────┼─────┼─────┤             ├─────┼─────┼─────┼─────┼─────┼─────┘
+  *       │     │     │     │     │     │             │     │     │     │     │     │
+  *       │     │     │     │     │     │             │     │     │     │     │     │
+  *       └─────┴─────┴─────┼─────┼─────┼─────┐ ┌─────┼─────┼─────┼─────┴─────┴─────┘
+  *                         │     │     │     │ │ DEL │     │     │
+  *                         │     │     │     │ │     │     │     │
+  *                         └─────┴─────┴─────┘ └─────┴─────┴─────┘
+  *       ┌─────┬─────┬─────┬─────┬─────┐             ┌─────┬─────┬─────┬─────┬─────┐
+  *       │     │     │     │     │ VOL │     NAV     │     │     │     │     │     │
+  *       │     │     │     │     │ UP  │    LAYER    │     │     │     │     │     │
+  * ┌─────┼─────┼─────┼─────┼─────┼─────┤             ├─────┼─────┼─────┼─────┼─────┼─────┐
+  * │     │ CTL │ CTL │ CMD │ CMD │ VOL │             │  ←  │  ↓  │  ↑  │  →  │CAPS │     │
+  * │     │ + ↓ │ + ↑ │ + ` │ +TAB│ DOWN│             │     │     │     │     │LOCK │     │
+  * └─────┼─────┼─────┼─────┼─────┼─────┤             ├─────┼─────┼─────┼─────┼─────┼─────┘
+  *       │     │     │     │     │ MUTE│             │     │     │     │     │     │
+  *       │     │     │     │     │     │             │     │     │     │     │     │
+  *       └─────┴─────┴─────┼─────┼─────┼─────┐ ┌─────┼─────┼─────┼─────┴─────┴─────┘
+  *                         │     │     │     │ │     │     │     │
+  *                         │     │     │     │ │     │     │     │
+  *                         └─────┴─────┴─────┘ └─────┴─────┴─────┘
+  *       ┌─────┬─────┬─────┬─────┬─────┐             ┌─────┬─────┬─────┬─────┬─────┐
+  *       │ F1  │ F2  │ F3  │ F4  │ F5  │     NUM     │ F6  │ F7  │ F8  │ F9  │ F10 │
+  *       │ F11 │ F12 │     │     │     │    LAYER    │     │     │     │     │     │
+  * ┌─────┼─────┼─────┼─────┼─────┼─────┤   ┌─────┐   ├─────┼─────┼─────┼─────┼─────┼─────┐
+  * │     │  1  │  2  │  3  │  4  │  5  │   │ TAP │   │  6  │  7  │  8  │  9  │  0  │     │
+  * │     │     │     │     │     │     │   │TAPx2│   │     │     │     │     │     │     │
+  * └─────┼─────┼─────┼─────┼─────┼─────┤   └─────┘   ├─────┼─────┼─────┼─────┼─────┼─────┘
+  *       │     │     │     │     │     │             │     │     │     │     │     │
+  *       │     │     │     │     │     │             │     │     │     │     │     │
+  *       └─────┴─────┴─────┼─────┼─────┼─────┐ ┌─────┼─────┼─────┼─────┴─────┴─────┘
+  *                         │     │     │     │ │     │     │     │
+  *                         │     │     │     │ │     │     │     │
+  *                         └─────┴─────┴─────┘ └─────┴─────┴─────┘
+  */
+```
```

Comment:
Shouldnt be here, and moved to a `keymaps/default/readme.md`

Reply:
[karn 1afcf5e783] Move keymap ascii diagram from root readme to keymaps/colemak/readme